docs/source/development-guide/style-guide/git-and-github-workflow.rst

```diff
@@ -59,26 +64,33 @@ well.*
 
    .. code-block:: bash
 
-       pip install .[dev]
-       pre-commit install
+       pip3 install pre-commit
+
+   If this results in errors, check the correct form of pip is being employed using ``which pip``.
 
 #. Retrieve the ``upstream`` ``dev`` branch:
 
    .. code-block:: bash
 
        git fetch upstream
-       git checkout -b dev upstream/dev
+       git checkout -b dev upstream-dev
+
+   To check this step, run ``git status``. The branch name should be ``upstream-dev``.
 
 #. Create a feature branch off of the ``dev`` branch to develop changes on. Branch names should be short but
    descriptive (e.g. ``update-codice-unit-tests``) and not too generic (e.g. ``bug-fix``, ``updates``). Consistent use
    of hyphens is encouraged.
 
+   Place the terminal in the file path of the fork you created in step one. Then, run the following command:
```

Comment:
This comment includes a suggested change - if this is something you like, you can commit it right from the UI using `commit suggestion`. Make sure you run `git pull` on your local branch whenever you do this. It can be a good way to make specific suggestions, but note that they are optional - if you don't like my suggestion, feel free to address the comment in another way (which we will cover on Monday in more depth.)

If someone is making a code suggestion, I highly suggest updating it locally and testing it before committing - it's really easy to make typos like missing parentheses or invalid methods when suggesting changes.